doc/en/api/clapi/objects/realtime_acknowledgement.rst

```diff
@@ -0,0 +1,145 @@
+=========================
+Real time Acknowledgement
+=========================
+
+Overview
+--------
+
+Object name: **RTACKNOWLEDGEMENT**
+
+Show host real time acknowledgement
+-----------------------------------
+
+In order to list available real time acknowledgement, use the **SHOW** action::
+You can use the value "HOST" to display all the acknowledgement::
+
+  [root@centreon ~]# ./centreon -u admin -p centreon -o RTACKNOWLEDGEMENT -a show -v "HOST;generic-host"
+  id;host_name;entry_time;author;comment_data;sticky;notify_contacts;persistent_comment
+  6;generic-host;2017/09/28 14:21;admin;'generic-comment';1;0;1
+
+Columns are the following :
+
+================================= ==================================================================================
+Column	                          Description
+================================= ==================================================================================
+Id	                              Id of the acknowledgement
+
+Host_name	                      Name of the host
+
+Entry_time                        Beginning of acknowledgement
+
+Author	                          Name of the author
+
+Comment_data                      Short description of the acknowledgement
+
+Sticky                            Acknowledgement will be maintained in case of a change of Not-OK status (0/1)
```

Comment:
```suggestion
Sticky                            Acknowledgement will be maintained in case of a change of Not-OK status (0/2)
```

Reply:
sticky need a value of 2 to work ;)
https://assets.nagios.com/downloads/nagioscore/docs/externalcmds/cmdinfo.php?command_id=39